buildSrc/build.gradle

```diff
@@ -19,7 +19,7 @@ dependencies {
     implementation 'de.undercouch.download:de.undercouch.download.gradle.plugin:5.0.1'
     implementation 'gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.7.2'
     implementation 'io.spring.dependency-management:io.spring.dependency-management.gradle.plugin:1.0.11.RELEASE'
-    implementation 'org.hidetake.swagger.generator:org.hidetake.swagger.generator.gradle.plugin:2.19.1'
+    implementation 'org.hidetake.swagger.generator:org.hidetake.swagger.generator.gradle.plugin:2.19.2'
```

Comment:
This may be a very trivial question, but how do you know when to upgrade a dependency, and how do you determine how far to upgrade it?

Reply:
I guess that's up for debate... in theory it's better to always be using the latest version. In practice, updating often leads to extra fixup or problem solving if the library has changed since the version we started with.

In this case, I tried updating to see if it helped address the `oneOf` weirdness. I'm not sure it did anything, but since it's newer than the previous version, I'm inclined to keep it anyway